code/obj/item/device/pda2/scanners.dm

```diff
@@ -184,6 +184,15 @@
 				for(var/datum/material_property/mat in A.material.getMaterialProperties())
 					var/value = A.material.getProperty(mat.id)
 					. += "• [mat.getAdjective(A.material)] ([value])<br>"
+				switch(A.material.getAlpha())
```

Comment:
Why aren't you processing like the rest with get adjective?

Reply:
Err, good point now that you mention it - from my limited knowledge though that'd need a datum/property that stores alpha that i can stick getAdjective on - should I do that?

Reply:
Ideally yes, you could preserve `getAlpha` as a wrapper proc to access the new opacity property's value though.